homeassistant/components/airzone/binary_sensor.py

```diff
@@ -95,8 +95,11 @@ def __init__(
     ) -> None:
         """Initialize."""
         super().__init__(coordinator, entry, system_zone_id, zone_data)
+
+        _id = entry.entry_id if (entry.unique_id is None) else entry.unique_id
```

Comment:
The `()` shouldn't be needed here

Reply:
Changed in 74573f9b29c8b13b657e1173e913ccf9b5c81d38